templates/decorator/name.decorator.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
 
+function <%= cameledName %>Decorator($delegate) 
+  // decorate the $delegate
+  return $delegate;
+}
+
+function <%= cameledName %>DecoratorConfig($provide) {
+    $provide.decorator('<%= cameledName %>', <%= cameledName %>Decorator($delegate) );
+}
+
 angular.module('<%= scriptAppName %>')
-  .config(function ($provide) {
-    $provide.decorator('<%= cameledName %>', function ($delegate) {
-      // decorate the $delegate
-      return $delegate;
-    });
-  });
+  .config(<%= cameledName %>DecoratorConfig);
```

Comment:
I think we can move the Decorator to a named function, but I think `<%= cameledName %>DecoratorConfig` is unnecessary, just inline it